source/board/microbit.c

```diff
@@ -18,8 +18,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-const char *board_id = "9900";
+#include "MKL26Z4.h"
```

Comment:
You don't have to change anything, but for future reference it is preferable to include `fsl_device_registers.h` instead of the specific register header by name.